server/tsconfig.json

```
@@ -20,5 +20,5 @@
     "module": "commonjs",
     "noEmit": false
   },
-  "include": ["*.ts"]
+  "include": ["*.ts", "index.ts"]
```

```suggestion
  "include": ["*.ts"]
```
This already includes all `.ts` files. Try restarting the TS Server or your VSCode